.github/workflows/appsignals-e2e-eks-test.yml

```diff
@@ -56,6 +56,12 @@ jobs:
             scripts/eks/appsignals/clean-app-signals.sh
           sparse-checkout-cone-mode: false
 
+      # TODO: remove this step next Monday and update validator with appropriate changes
+      - name: Temporary override addon version
+        working-directory: enablement-script/scripts/eks/appsignals
+        run: |
+          sed -i '61 a--addon-version v1.2.2-eksbuild.1 \\' enable-app-signals.sh
```

Comment:
Not at all ideal using exact line number, would be better to find `--region ${REGION}` and replace it with `--region ${REGION} --addon-version v1.2.2-eksbuild.1`, i.e. append the option